variants/fcos-multiarch/kustomization.yaml

```diff
@@ -21,10 +21,26 @@ patches:
           dockerStrategy:
             buildArgs:
               - name: "TAGS"
-                value: "fcos"
+                value: "okd fcos hyperkube"
```

Comment:
not sure what the `not-used` BC does, but on the installer, `okd` and `fcos` are synonymous, with `okd` now better being replaced by either `fcos` or `scos`.

Reply:
```suggestion
                value: "fcos hyperkube"
```